README.md

```diff
@@ -1 +1,43 @@
-# silverstripe-tx-translator
+# silverstripe tx translator
+
+Module to assist with transifex integration. Works with Silverstripe 4.12+ and 5.0+.
+
+## Operations
+
+- Pull latest translations from transifex and merge them into yml/json/js translation files
+- Run i18nTextCollectorTask on a local website
+- Push updated source strings to transifex (optional)
+- Create github pull-requests with file changes
+
+## Requirements
+
+Transifex cli client (tx) - [https://developers.transifex.com/docs/cli](https://developers.transifex.com/docs/cli) configured API key from transifex.
+
+You must use the go version at least version 1.6+ and not the old python version.
+
+Create a new classic github api token in [github token settings](https://github.com/settings/tokens) with all checkboxes unticked except for `public_repo`. This is required to create pull-requests.
```

Comment:
I tried for a long time to get a finely grained token going, but I just couldn't work out the required permissions.  I had 'Actions', 'Contents', 'Pull-request' all set to read & write, still got a response back saying I had the wrong permissions.

Note: there used to be code in here to fallback to use the composer API token - but this is a bad idea, my composer token failed because I lacked permissions. I think saying that you must create a new token with the correct permissions and then delete it in the readme is the best way to reduce confusion for why pull-request don't get created.